sqlglot/expressions.py

```diff
@@ -4742,6 +4862,41 @@ def delete(
     return delete_expr
 
 
+def insert(
+    expression: ExpOrStr,
+    into: ExpOrStr,
+    columns: t.Optional[t.Sequence[ExpOrStr]] = None,
+    overwrite: t.Optional[bool] = None,
+    dialect: DialectType = None,
+    copy: bool = True,
+    **opts,
+) -> Insert:
+    """
+    Builds an INSERT statement.
+
+    Example:
+        >>> insert("VALUES (1, 2, 3)", "tbl").sql()
+        'INSERT INTO tbl VALUES (1, 2, 3)'
+
+    Args:
+        expression: the sql string or expression of the INSERT statement
+        into: the tbl to insert data to.
+        columns: optionally the table's column names.
+        overwrite: whether to INSERT OVERWRITE or not.
+        dialect: the dialect used to parse the input expressions.
+        copy: whether or not to copy the expression.
+        **opts: other options to use to parse the input expressions.
+
+    Returns:
+        Insert: the syntax tree for the INSERT statement.
+    """
+    return (
```

Comment:
just implement all thaat stuff here and remove the Insert().insert.into, i'm not really sure i buy into that api

Reply:
Yeah.. I mimicked the other builders without thinking much about it, but what you're saying makes sense.